### PR TITLE
Updating this neuron a bit

### PR DESCRIPTION
**Pull request by daylicron**
Hey,
I noticed the following two things, when I started to use this neuron:
- install.yml: Version bum of requests as kalliope requires =>13.x
- tasker_autoremote.py: Add option for password if someone wants to secure autoremote in addition with a password

Cheers
Dennis

### CODE REVIEW

**Comment by Sispheor** [comment]
You can remove the "version line for the install. It will take the last version available.

File: tasker_autoremote.py [hunk 1]
```diff
@@ -15,12 +15,14 @@ def __init__(self, **kwargs):
...
         self.message = kwargs.get('message', None)
+        self.password = kwargs.get('password', None)
 
         # check parameters
         if self._is_parameters_ok():
             # create the payload
             data = {'key': self.key,
-                    'message': self.message}
+                    'message': self.message,
+                    'password': self.password}
```
Comment (Sispheor) [inline tasker_autoremote.py]: If you set it in the payload, then it become mandatory and will be set to None for those who havn't set it in the neuron. And I presume that the API will fail if the received password key is empty.

**Review by Sispheor — CHANGES_REQUESTED** [review]

File: tasker_autoremote.py (hunk 1 above)
Comment (daylicron) [reply tasker_autoremote.py]: I checked this before and it works when you leave password empty. Maybe someone can confirm this.

Comment (Sispheor) [reply tasker_autoremote.py]: It's anyway better to only add it when provided. Use the update method on the data dict.

**Comment by Sispheor** [comment]
Missing doc update also.

**Comment by daylicron** [comment]
@Sispheor : sry, I've been quite busy. I've built in your recommendations. Anything else missing here?

**Review by Sispheor — APPROVED** [review]

**Pull request merged**